Edge10.CouchDb.Client.Tests/packages.config

```diff
@@ -8,4 +8,5 @@
   <package id="NUnit.Extension.NUnitProjectLoader" version="3.5.0" targetFramework="net452" />
   <package id="NUnit.Extension.TeamCityEventListener" version="1.0.2" targetFramework="net452" />
   <package id="NUnit.Extension.VSProjectLoader" version="3.5.0" targetFramework="net452" />
+  <package id="NUnit3TestAdapter" version="3.7.0" targetFramework="net452" />
```

Comment:
Should it be consistent with other dependencies and be 3.5 version, what do you think?